soapui/src/main/java/com/eviware/soapui/impl/wsdl/testcase/WsdlProjectRunner.java

```diff
@@ -128,6 +161,7 @@ private void runSequential(WsdlProject project, WsdlProjectRunContext runContext
     }
 
     private void updateStatus() {
+        activeRunners.clear();

```

Comment:
I suggest not to mix managing runners with updating status. My proposal is 
```
if (runCount.get() == 0) {
  activeRunners.clear();
  updateStatus();
}
```
this code resides below inside InternalTestSuiteRunListener